test/cpp/test_aten_xla_tensor_1.cpp

```diff
@@ -1400,6 +1400,24 @@ TEST_F(AtenXlaTensorTest, TestDropoutInPlace) {
   ExpectCounterChanged("xla::bernoulli", cpp_test::GetIgnoredCounters());
 }
 
+TEST_F(AtenXlaTensorTest, TestNativeDropout) {
+  torch::Tensor a = torch::rand({17, 21}, torch::TensorOptions(torch::kFloat));
+  ForEachDevice([&](const torch::Device& device) {
+    torch::Tensor xla_a = CopyToDevice(a, device);
+    auto [xla_b_val, xla_b_mask] = torch::native_dropout(xla_a, 0.1, /*train=*/true);
+    double prob =
+        static_cast<double>(xla_b_val.cpu().ne(0.0f).sum().item().toDouble()) /
+        a.numel();
+    EXPECT_GT(prob, 0.86);
+    EXPECT_LT(prob, 0.94);
```

Comment:
We also need to check the correctness of mask, we can check `prob_activation == prob_mask`

Also we need to check the output dtype of activation and mask
```
EXPECT_EQ(xla_b_val.scalar_type(), torch::kFloat);
EXPECT_EQ(xla_b_mask.scalar_type(), torch::kBool);
```

Reply:
Cool! updated with 
```
EXPECT_GT(prob, 0.06);
EXPECT_LT(prob, 0.14);
```
And now the condition passed.